### PR TITLE
xdman: Fix shortcut

### PR DESCRIPTION
**Pull request by candrapersada**
Closes #XXXX

Relates to #XXXX

- [ ] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/xdman.json [hunk 1]
```diff
@@ -21,6 +23,7 @@
         "8117f286f9de1e57f4551676d8f0ab3e2e990444744b595f7cc4cb9e01f51c7c",
         "7da503e3d399b804b19b05eaff27bcace3915b12da863161af68d13ff8050bf0"
     ],
+    "pre_install": "Set-Content \"$dir\\xdman.bat\" '@start javaw.exe -jar xdman.jar' -Encoding Ascii",
```
[the remark this answers is not in the document]
Comment (candrapersada) [reply bucket/xdman.json:26]: for run xdman.jar from java portable

[the remark this answers is not in the document]
Comment (HUMORCE) [reply bucket/xdman.json:26]: currently, only shortcut is not working properly, right?

scoop has a built-in method to create batch file(cmd) for jar pack.

Comment (HUMORCE) [inline bucket/xdman.json:26]:
```suggestion
    "pre_install": "Set-Content $dir\\shortcut.cmd \"@start javaw -jar $dir\\xdman.jar\" -Encoding ascii",
```

**Comment by HUMORCE** [comment]
the shortcut also runs the app properly, now.

i have never used this package, so I did not notice the shortcut issue in prev PR.

**Pull request merged**